ansible/roles/keycloak/templates/values.yaml

```diff
@@ -0,0 +1,126 @@
+replicaCount: "{{ keycloak_replica_count }}"
+image:
+  repository: "{{ keycloak_image_repository }}"
+  tag: "{{ keycloak_image_tag }}"
+  pullPolicy: "{{ keycloak_image_pull_policy }}"
+  pullSecrets:
+    {{ keycloak_image_pull_secrets | to_nice_yaml | indent(4) | trim  }}
+
+auth:
+  existingSecret: "{{ keycloak_auth_existing_secret }}"
+  secretName: "{{ keycloak_auth_secret_name }}"
+
+  admin_password: "{{ keycloak_auth_admin_password }}"
+  keycloak_rs256_privateKey: "{{ keycloak_auth_rs256_private_key }}"
+  keycloak_rs256_certificate: "{{ keycloak_auth_rs256_certificate }}"
+  keycloak_adminSecret: "{{ keycloak_auth_admin_secret }}"
+  keycloak_clientSecret: "{{ keycloak_auth_client_secret }}"
+  keycloak_serviceClientCertificate: "{{ keycloak_auth_service_client_certificate }}"
+
+importRealms: "{{ keycloak_import_realms }}"
+
+command:
+  {{ keycloak_command | to_nice_yaml | indent(2) | trim }}
+extraArgs:
+  {{ keycloak_extra_args | to_nice_yaml | indent(2) | trim  }}
+extraVolumes:
+  {{ keycloak_extra_volumes | to_nice_yaml | indent(2) | trim }}
+extraVolumeMounts:
+  {{ keycloak_extra_volume_mounts | to_nice_yaml | indent(2) | trim }}
+
+extraConfig:
+  name: "{{ keycloak_extra_config_name }}"
+  create: "{{ keycloak_extra_config_create }}"
+  data:
+    {{ keycloak_extra_config_data | to_nice_yaml | indent(4) | trim  }}
+  external:
+    {{ keycloak_extra_config_external  | to_nice_yaml | indent(4) | trim }}
+
+extraSecret:
+  name: "{{ keycloak_extra_secret_name }}"
+  create: "{{ keycloak_extra_secret_create }}"
+  data:
+    {{ keycloak_extra_secret_data | to_nice_yaml | indent(4) | trim  }}
+  external:
+    {{ keycloak_extra_secret_external | to_nice_yaml | indent(4) | trim }}
```

Comment:
Validate if is it need to set and Itaret over the list, don't do this the helm way